flink-kubernetes-operator/pom.xml

```diff
@@ -413,6 +425,22 @@ under the License.
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>depend-on-okhttp4</id>
+            <activation>
+                <property>
+                    <name>fabric8.httpclinent.impl</name>
+                    <value>okhttp</value>
+                </property>
+            </activation>
+            <dependencies>
+                <dependency>
+                    <groupId>com.squareup.okhttp3</groupId>
+                    <artifactId>okhttp</artifactId>
+                    <version>${okhttp.version}</version>
+                </dependency>
```

Comment:
By default `kubernetes-httpclient-okhttp` pulls in OkHttp 3.X which is reported to have problems with Kubernetes IPV6 stacks e.g. https://issues.apache.org/jira/browse/FLINK-31928